ledger-tool/src/main.rs

```diff
@@ -2143,6 +2149,8 @@ fn main() {
                     filler_accounts_config,
                     skip_rewrites: matches.is_present("accounts_db_skip_rewrites"),
                     ancient_append_vecs: matches.is_present("accounts_db_ancient_append_vecs"),
+                    skip_initial_hash_calc: matches
+                        .is_present("accounts_db_skip_initial_hash_calc_arg"),
```

Comment:
I think we need to drop the _arg in the string here.

Reply:
yes. good catch. and I should not have abbreviated the argument.